lab-david/lib/basic-auth-middleware.js

```diff
@@ -0,0 +1,25 @@
+'use strict';
+
+const createError = require('http-errors');
+const debug = require('debug')('cfgram:basic-auth-middleware');
+
+module.exports = function(req, res, next){
+  debug('basic auth');
+
+  var authHeader = req.headers.authorization;
```

Comment:
Brian digs var, I'm all about const if I'm not reassigning it, which you aren't here.